extensions/rich_text_components/Link/directives/oppia-noninteractive-link.component.ts

```diff
@@ -62,8 +62,8 @@ export class NoninteractiveLink implements OnInit, OnChanges {
     let untrustedUrl = encodeURI(this.htmlEscaperService.escapedJsonToObj(
       this.urlWithValue) as string);
     if (
-      untrustedUrl.startsWith('http://') &&
-      untrustedUrl.startsWith('https://')
+      !untrustedUrl.startsWith('http://') &&
+      !untrustedUrl.startsWith('https://')
```

Comment:
I added `!` since `https` should be added to the link provided by the user if the link does not start with `http` or `https`.

The current change checks if either `http` or `https` is present. If both are not then is `https` is added to the link provided by the user.